extensions/arc/src/new-note.ts

```diff
@@ -0,0 +1,31 @@
+import { showHUD } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+
+function runAppleScriptAction() {
+  return runAppleScript(`
+      tell application "Arc"
+      if (count of windows) is 0 then
+        make new window
+      end if
+      activate
+    end tell
+    delay (0.5)
+    tell application "Arc"
+      activate
+    end tell
+
+    tell application "System Events"
+      tell process "Arc"
+        click menu item "New Note" of menu "File" of menu bar 1
+      end tell
+    end tell
+  `);
+}
+
+export default async function command() {
+  try {
+    await runAppleScriptAction();
+  } catch {
+    await showHUD("❌ Failed opening a new note");
+  }
+}
```

Comment:
Should this file be here if the previous update removed `New Note` command?

Reply:
No, it shouldn't. Tbh, I don't even know how this file ended up there when I forked the extension. 🤔